pandas/tests/test_series.py

```diff
@@ -203,6 +203,28 @@ def f():
                 s.dt.hour[0] = 5
             self.assertRaises(com.SettingWithCopyError, f)
 
+    def test_strftime(self):
+        # GH 10086
+        s = Series(date_range('20130101', periods=5))
+        result = s.dt.strftime('%Y/%m/%d')
```

Comment:
add something with say millseconds, and show various formats for that (e.g. very common to eliminate hh:mm:dd......) and things like that